1906-10-22.xml

```diff
@@ -0,0 +1,5626 @@
+                    <div type="advert" xml:id="deg-ad-cun02">
+                        <head>Cunard Line.</head>
+                        <p>
+                            <measure type="indexNo">19-1-907</measure>
+                        </p>
+                        <p>Frequent sailings of cargo steamers from Alexandria to Liverpool. Through
+                            Bills of Lading to towns in the Interior and to the United States
+                            ports.</p>
+                        <p>Sailings of ROYAL MAIL passenger steamers from Liverpool to New-York and
+                            Boston on Saturdays and Tuesdays.</p>
+                        <p>Regular twin-screw passenger service between Trieste, Fiume, Naples, and
+                            New-York.</p>
+                        <p>All passenger steamers fitted with Marconi's wireless telegraphy. For
+                            through tickets from Egypt and particulars, apply to the Agents:</p>
+                        <p>Rodacanachi &amp; Co., Alexandria; Nicolas Kirgis, Cairo; R. Broadbent,
+                            Port Said.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-pss01">
+                        <head>PORT SAID-SAVOY HOTEL.</head>
+                        <p>NEW FIRST-CLASS HOTEL, OVERLOOKING THE HARBOUR &amp; OPPOSITE CUSTOM
+                            HOUSE</p>
+                        <p>Open all the year round. — Well-appointed Bar.</p>
+                        <p>MODERATE CHARGES. SPECIAL TERMS FOR RESIDENTS 1190A2-5</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-nkh01">
+                        <head>NEW KHEDIVIAL HOTEL, ALEXANDRIA.</head>
+                        <p>First-class Hotel. Situated in Rosetta Avenue, the finest quarter in the
+                            Town. Two mintes from Railway Station. Close to Conservatory and the
+                            Opera House. Lift. Electric Light Throughout. Perfect Sanitary
+                            Arrangements. Magnificent Ball, Reception, Reading, and Music Rooms. Bar
+                            and Smoking Room.</p>
+                        <p> HENRI CHAMOULLEAU, Proprietor.</p>
+                        <p>45</p>
+                        <p>FINE TERRACE ON THE AVENUE. - SPLENDID GARDEN. - OMNIBUS MEET ALL TRAINS
+                            AND STEAMERS. <measure type="indexNo">28-26</measure></p>
+                    </div>
+                    <cb n="3"/>
+                    <div type="advert" xml:id="deg-ad-aan03">
+                        <head>Hamburg &amp; Anglo-American Nile Co.</head>
+                        <p>Weekly departure during Winter Season by the<lb/> Luxurious First Class
+                            Tourist Steamers VICTORIA, PURITAN &amp; MAYFLOWER.<lb/> Regular weekly
+                            Departures to the SE
```

Comment:
each of the paragraphs here should be an item within the prepaid advertisement section, just above.